docs/index.md

```diff
@@ -0,0 +1,40 @@
+# Auth0 Provider
```

Comment:
Pages should have [frontmatter](https://github.com/hashicorp/terraform-website#yaml-frontmatter) on the first lines so they can be picked up by terraforms documentation tooling. 

For example, this page could start with the following:

```
---
layout: "auth0"
page_title: "Provider: Auth0"
description: |-
  The Auth0 provider is used to interact with Auth0 applications and APIs.
---

# Auth0 Provider

The Auth0 provider is...
```

If you need my help with this, please let me know. I'd be happy to assist.